File: src/main/java/com/scrumwall/service/column/ColumnServiceImpl.scala
package com.scrumwall.service.column

import com.scrumwall.util.debug.LogsToLog4J
import com.scrumwall.dao.column.ColumnDao
import com.scrumwall.domain.Column
import java.util.List

class ColumnServiceImpl extends ColumnService {

  var columnDao: ColumnDao = _
  def setColumnDao(columnDao: ColumnDao){
    this.columnDao = columnDao
  }
  
  override def getColumns : List[Column] = {
    debug("Getting all columns")
    this.columnDao.getColumns
  }
  
  override def save(column: Column): Column = {
    columnDao save column 
    
    //this.columnDao.updateOrders()
    
    
    column
  }
}

File: src/main/java/com/scrumwall/gwt/server/service/ColumnServiceImpl.java
package com.scrumwall.gwt.server.service;

import java.util.List;

import org.springframework.beans.factory.annotation.Autowired;

import com.scrumwall.gwt.RpcFacade;
import com.scrumwall.gwt.client.ColumnService;
import com.scrumwall.gwt.server.dao.ColumnDao;
import com.scrumwall.gwt.shared.ColumnDTO;

@RpcFacade
public class ColumnServiceImpl implements ColumnService {

	@Autowired
	private ColumnDao columnDao;
	
	public List<ColumnDTO> getColumns(Integer sprintId){
		return columnDao.getColumns(sprintId);
	}
	
}
